packages/clay-date-picker/src/DayNumber.tsx

```diff
@@ -15,24 +15,30 @@ interface IProps {
 	onClick: (date: Date) => void;
 }
 
+const KEYS = {
+	SPACE: ('Spacebar' || ' '),
```

Comment:
@diegonvs I think that this will not work to check both, we have to make explicit in the `if` conditional below. What we are doing here is conditional if `Spacebar` does not exist which will never be true.

Reply:
TS39 proposal: New binary operator, `││` (2 x Unicode U+2502, "Box Drawings Light Vertical"), which represents the quantum superposition of the left and right side operands.

Reply:
🤔 Hmm

Reply:
I was minding https://en.wikipedia.org/wiki/Distributive_property 🤔